src/lib_ccx/ccx_decoders_isdb.c

```diff
@@ -182,7 +182,7 @@ struct ISDBPos{
 
 struct ISDBText
 {
-	char *buf;
+	unsigned char *buf; // All the buffers in the project are usigned but it. Why?
```

Comment:
Probably because it's more convenient in this case (not sure). I see you cast it to char * later...